src/strings_internal.h

```diff
@@ -130,4 +132,10 @@ class StringBuilder {
 	}
 };
 
+void GetStringWithArgs(StringBuilder &builder, StringID string, StringParameters *args, uint case_index = 0, bool game_script = false);
+
+/* Do not want to leak the StringBuilder to everywhere. */
```

Comment:
This was a mental note? As in: `I do not want to leak ..`. Or was there another intention here? Or should `want to` be removed?

Sorry, comment puzzled me for a while.

Reply:
It's more a mental note for other developers/reviewers, that 'strings_internal.h' should not be included in other headers. I removed the 'want to'.